src/olympia/tags/tests/test_views.py

```diff
@@ -0,0 +1,25 @@
+from olympia.amo.tests import APITestClient, TestCase, reverse_ns
+
+from ..models import Tag
+
+
+class TestStaticCategoryView(TestCase):
+    client_class = APITestClient
+
+    def setUp(self):
+        super().setUp()
+        self.url = reverse_ns('tag-list')
+
+    def test_basic(self):
+        response = self.client.get(self.url)
+        assert response.status_code == 200
+        tags = list(Tag.objects.all())
```

Comment:
Will there be any existing tags in the test environment when this is run?

Reply:
yeah there's a migration that's run while setting up the test db that adds all tags that were defined a few months back.

Reply:
Would it be worth asserting that `len(response.data) > 0`, just to make me feel more secure about this (as well as any future reader of the code)? I added the comment because I notice that we often generate test data in the test itself, but we're not doing that here.